src/main-output.cpp

```diff
@@ -83,13 +83,20 @@ void main_output_start()
 			obs_log(LOG_INFO,
 				"main_output_start: successfully started NDI main output '%s'",
 				QT_TO_UTF8(context.ndi_name));
+			Config::Current()->OutputLastError = QString("");
 		} else {
 			auto error = obs_output_get_last_error(context.output);
+			Config::Current()->OutputLastError =
+				"Last Error: Unsupported color format " +
```

Comment:
Can we localize this error message? Let's avoid this to be hard coded if visible to end-users.